Project.toml

```diff
@@ -13,17 +13,22 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 FixedPointNumbersStatisticsExt = "Statistics"
 
 [compat]
+Aqua = "0.8"
+Documenter = "0.27, 1"
```

Comment:
We can drop the test dependency on Documenter.jl by adding documentation with Documenter.jl I will open a PR for that later.